packages/http-client-java/generator/http-client-generator-clientcore-test/src/test/java/server/path/multiple/MultipleClientTest.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package server.path.multiple;
+
+import org.junit.jupiter.api.Test;
+
+class MultipleClientTest {
+
+    MultipleClient client = new MultipleClientBuilder().endpoint("http://localhost:3000").buildClient();
+
+    @Test
+    void noOperationParams() {
+        client.noOperationParams();
+    }
```

Comment:
Please update these following the common patten (public class, private final var, public method).
If azure part is same, please also update them as well.